packages/components/src/tooltip/test/index.js

```diff
@@ -217,8 +218,8 @@ describe( 'Tooltip', () => {
 				</Tooltip>
 			);
 
-			const eventCatcher =
-				container.getElementsByClassName( 'event-catcher' )[ 0 ];
+			// eslint-disable-next-line testing-library/no-container
+			const eventCatcher = container.querySelector( '.event-catcher' );
```

Comment:
Using the same query as above. This element is not accessible by design, so querying it that way makes sense.